docs/README.md

```diff
@@ -28,6 +28,8 @@ As a *developer*, designate the appropriate workload type to execute your compon
 #### [Create Component from Scratch](how-to/create_component_from_scratch.md)
 Build a component from source code to use for testing.
 
+#### [Manage complicated parameters](how-to/manage_complicated_parameters.md)
```

Comment:
"Manage complicated parameters" is a bit confusing, it may read like OAM will handle "complicated parameters" to some extend.

Sth like: `Using Helm/Kustomize to manage OAM yamls` will be clear.

Reply:
yeah, I'll change It